env_support/cmake/version.cmake

```diff
@@ -0,0 +1,4 @@
+set(LVGL_VERSION_MAJOR 8 PARENT_SCOPE)
+set(LVGL_VERSION_MINOR 0 PARENT_SCOPE)
+set(LVGL_VERSION_PATCH 3 PARENT_SCOPE)
+set(LVGL_VERSION_INFO "dev" PARENT_SCOPE)
```

Comment:
Could you also please add
```
set(LVGL_VERSION "${LVGL_VERSION_MAJOR}.${LVGL_VERSION_MINOR}.${LVGL_VERSION_PATCH}")
```

This way all CMake version compares can be used https://cmake.org/cmake/help/latest/command/if.html#version-comparisons